include/unifex/when_all.hpp

```diff
@@ -63,8 +63,8 @@ template <
     typename... Rest>
 struct _operation_tuple<Index, Receiver, First, Rest...>::type
   : operation_tuple<Index + 1, Receiver, Rest...> {
-  template <typename Parent>
-  explicit type(Parent& parent, First&& first, Rest&&... rest)
+  template <typename First2, typename Parent>
+  explicit type(Parent& parent, First2&& first, Rest&&... rest)
```

Comment:
Why was this change necessary?
Shouldn't the sender be perfectly forwarded through to this constructor?

Reply:
I reverted this and it compiles locally. Thanks!